openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/GlossaryVersionPage.spec.js

```diff
@@ -139,7 +139,7 @@ describe('Glossary and glossary term version pages should work properly', () =>
 
     cy.get('[data-testid="version-button"]').contains('0.2');
 
-    addOwner(OWNER);
+    addOwner(OWNER, 'glossary-right-panel-owner-link');
```

Comment:
Can we `glossary-right-panel-owner-link` this to constant, so in future, we have to update at one place only?